bindings/pulp/bindings/server.py

```diff
@@ -223,7 +225,7 @@ def _build_url(self, path, queries=()):
                     of self.path_prefix
         """
         # build the request url from the path and queries dict or tuple
-        if not path.startswith(self.path_prefix):
+        if not path.startswith(self.path_prefix) and not ignore_prefix:
```

Comment:
Since these have no docblocks, I did not want to add an incomplete/inaccurate docblock, and I also didn't want to spend the time to investigate each of these params just to add it.